[2023-12-13T02:02:41.788+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_first_dag_v3.third_task scheduled__2023-12-12T00:00:00+00:00 [queued]>
[2023-12-13T02:02:41.798+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_first_dag_v3.third_task scheduled__2023-12-12T00:00:00+00:00 [queued]>
[2023-12-13T02:02:41.798+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2023-12-13T02:02:41.815+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): third_task> on 2023-12-12 00:00:00+00:00
[2023-12-13T02:02:41.820+0000] {standard_task_runner.py:57} INFO - Started process 215 to run task
[2023-12-13T02:02:41.824+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'my_first_dag_v3', 'third_task', 'scheduled__2023-12-12T00:00:00+00:00', '--job-id', '94', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmp406xxlhe']
[2023-12-13T02:02:41.826+0000] {standard_task_runner.py:85} INFO - Job 94: Subtask third_task
[2023-12-13T02:02:41.879+0000] {task_command.py:416} INFO - Running <TaskInstance: my_first_dag_v3.third_task scheduled__2023-12-12T00:00:00+00:00 [running]> on host 6bd2d0daa4c2
[2023-12-13T02:02:41.996+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='vuhta' AIRFLOW_CTX_DAG_ID='my_first_dag_v3' AIRFLOW_CTX_TASK_ID='third_task' AIRFLOW_CTX_EXECUTION_DATE='2023-12-12T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-12-12T00:00:00+00:00'
[2023-12-13T02:02:41.997+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-12-13T02:02:41.998+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo hey this is the third task executed after task 1 and at the same time with task 2']
[2023-12-13T02:02:42.011+0000] {subprocess.py:86} INFO - Output:
[2023-12-13T02:02:42.014+0000] {subprocess.py:93} INFO - hey this is the third task executed after task 1 and at the same time with task 2
[2023-12-13T02:02:42.015+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-12-13T02:02:42.040+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=my_first_dag_v3, task_id=third_task, execution_date=20231212T000000, start_date=20231213T020241, end_date=20231213T020242
[2023-12-13T02:02:42.076+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-12-13T02:02:42.092+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
